website/content/docs/platform/k8s/consul-to-raft.mdx

```diff
@@ -0,0 +1,361 @@
+    - `replicas: 1`
+      - This setting is temporary for the purposes of the migration.  A new Vault `StatefulSet` with one replica to confirm the init container completed the migration and unseal Vault using the new storage backend.
+
+1. Apply this configuration.
+
+  ```shell-session
+  $ helm install vault hashicorp/vault -f vault-migration-values.yml
+  ```
+  
+1. Review migration logs and unseal Vault
+  
+  ```shell-session
+  $ kubectl logs vault-0 -c vault-server-migration
+  ```
+  
+  ```shell-session
+  $ kubectl exec -it vault-0 -- vault operator unseal
+  Key                     Value
+  ---                     -----
+  Seal Type               shamir
+  Initialized             true
+  Sealed                  false
+  Total Shares            1
+  Threshold               1
+  Version                 1.14.8+ent
+  Build Date              2023-12-05T01:49:39Z
+  Storage Type            raft
+  Cluster Name            vault-cluster-68870bf8
+  Cluster ID              cd18c692-f2e3-77a5-fba3-28f06f41f375
+  HA Enabled              true
+  HA Cluster              https://vault-0.vault-internal:8201
+  HA Mode                 active
+  Active Since            2024-04-10T04:20:23.707098402Z
+  Raft Committed Index    157
+  Raft Applied Index      157
+  Last WAL                55
+  ```
+
+1. Update Vault Helm deployment with Raft storage configuration
+
+  ```shell-session
+  $ cat > vault-raft-values.yml <<EOF
+  global:
+    enabled: false
+  server:
+    enabled: true
+    image:
+      repository: hashicorp/vault-enterprise
+      tag: 1.14.8-ent
+    enterpriseLicense:
+      secretName: vault-license
+      secretKey: vault.hclic
+    dataStorage:
+      enabled: true
+      size: "1Gi"
+    ha:
+      enabled: true
+      replicas: 5
+      raft:
+        enabled: true
+        config: |
+          ui = true
+          service_registration "kubernetes" {}
+
+          listener "tcp" {
+            address = ":8200"
+            cluster_address = ":8201"
+            tls_disable = 1
+          }
+
+          storage "raft" {
+            path = "/vault/data"
+            retry_join {
+              auto_join_scheme = "http"
+              auto_join = "provider=k8s"
+            }
+          }
+  EOF
+  ```
+  **Configuration notes**
+    - `replicas: 5`
+      - Upgrade the Helm deployment in place using the final Raft storage configuration, removing the `extraInitContainer` and storage migration `ConfigMap`, and increasing the number of replicas.  The `retry_join` parameters used by the new Vault server replicas to automatically join the cluster.
+
+1. Apply the configuration.
+
+  ```shell-session
+  $ helm upgrade vault hashicorp/vault -f vault-raft-values.yml
+  ```
+  
+1. Unseal Vault and confirm the Raft peers have formed a quorum.
+
+  ```shell-session
+  $ for i in {1..4} ; do kubectl exec -it vault-0 -- vault operator unseal ; done
+  ```
+
```

Comment:
````suggestion
1. Unseal Vault.

  ```shell-session
  $ for i in {1..4} ; do kubectl exec -it vault-0 -- vault operator unseal ; done
  ```

1. Confirm the Raft peers have formed a quorum.

````